src/lib/components/input/input.md

```diff
@@ -1,65 +1,79 @@
 ```js
 initialState = {
-  value: 'Hey! I am an input',
+  value: ""
 };
 
 function updateValue(name, value) {
   setState({ [name]: value });
 }
 
 <Input
-  placeholder="Name"
-  value={state.value}
-  required={true}
   name="value"
+  value={state.value}
+  onChangeFunction={updateValue}
+  label="Placeholder input"
 />;
 ```
 
-```jsx inside Markdown
-import React from 'react';
-import { validateEmail } from '../../utils/validation';
-
+```js
 initialState = {
-  value: 'You need to enter an email here!',
+  value: ""
 };
 
 function updateValue(name, value) {
   setState({ [name]: value });
 }
 
 <Input
-  placeholder="Email"
-  value={state.value}
-  required={true}
   name="value"
+  value={state.value}
   onChangeFunction={updateValue}
-  validations={[validateEmail]}
+  label="Placeholder input"
+  helperText="My super helper text"
 />;
 ```
 
 ```js
 initialState = {
-  value: 'Hey! I am a light themed input.',
+  required: ""
 };
 
 function updateValue(name, value) {
   setState({ [name]: value });
 }
 
 <Input
-  placeholder="Light theme"
-  value={state.value}
-  required={true}
-  name="value"
+  name="required"
+  value={state.required}
   onChangeFunction={updateValue}
-  theme="light"
+  label="Required input"
+  required
 />;
 ```
 
 ```js
+initialState = {
+  error: ""
+};
+
+function updateValue(name, value) {
+  setState({ [name]: value });
+}
+
 <Input
-  value="I am read only. Don't try to change me!"
```

Comment:
This needs to be on a t-shirt!!!  `value="I am read only. Don't try to change me!"`